server/api/reports.py

```diff
@@ -441,6 +446,36 @@ def round_rows(election: Election):
             total_choice_votes.update(non_cvr_choice_vote)
             total_choice_votes.update(cvr_choice_votes)
 
+        if election.audit_type == AuditType.BATCH_COMPARISON:
+            distinct_batches = (
+                Batch.query.join(SampledBatchDraw)
+                .join(Round)
+                .join(Jurisdiction)
+                .filter_by(election_id=election.id)
+                .filter(SampledBatchDraw.round_id == round.id)
+                .all()
+            )
+            num_distinct_batches = len(distinct_batches)
+            num_distinct_ballots = sum(batch.num_ballots for batch in distinct_batches)
+
+            # get the reported votes aggregated across batches
+            reported_results: dict = {choice.id: 0 for choice in contest.choices}
+            for batch in distinct_batches:
+                reported_results_for_batch = (
+                    batch.jurisdiction.batch_tallies[batch.name][contest.id]
+                    if contest.id in batch.jurisdiction.batch_tallies[batch.name]
+                    else None
+                )
```

Comment:
Consider: `batch.jurisdiction.batch_tallies[batch.name].get(contest.id)`, which will return `None` if the key is missing from the dict